helm-chart/renku-notebooks/templates/clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if and .Values.rbac.create .Values.userSessionPersistentVolumes.enabled -}}
```

Comment:
Should we try to start giving more indicative names to the templates? We now have 

- role.yaml
- rolebinding.yaml
- clusterrole.yaml (which containts the clusterrolebinding)

I already see myself being confused about where to find what. How about:

- /templates/rbac/image-pull-secrets.yaml
- /templates/rbac/session-pvc.yaml

both containing the (cluster)role and -binding? Maybe also name the actual k8s resources accordingly?

Reply:
thanks @ableuler - I've made a template called `session-pv-rbac.yaml` that includes a custom role/rolebinding and the cluster role/rolebinding.